src/Language/GraphQL/Draft/Parser.hs

```diff
@@ -479,3 +489,41 @@ between open close p = tok open *> p <* tok close
 -- `empty` /= `pure mempty` for `Parser`.
 optempty :: Monoid a => Parser a -> Parser a
 optempty = option mempty
+
+-- | Parses strings delimited by triple quotes. 
+-- http://spec.graphql.org/June2018/#sec-String-Value
+blockString :: Parser Text
+blockString = do
+  _ <- triplequotes <?> "opening triple quotes"
+  str <- T.lines . T.pack <$> AT.manyTill AT.anyChar triplequotes <?> "the body of a triple quoted string"
+  let !cleanLines = dropEmptyLines str
```

Comment:
What does this bang do? It certainly won't force the entire list to normal form.